OpenBSD doesn't have GNU ffsll() but both gcc and clang have it as builtin

Index: src/libutil/printf.c
--- src/libutil/printf.c.orig
+++ src/libutil/printf.c
@@ -42,6 +42,8 @@
 #include "str_util.h"
 #include "contrib/fpconv/fpconv.h"
 
+#define ffsll   __builtin_ffsll
+
 /**
  * From FreeBSD libutil code
  */
